assets/environment-interpreter/activate/activate

```diff
@@ -263,9 +263,11 @@ if [ "$_FLOX_ENV_ACTIVATION_MODE" = "build" ]; then
     echo "build mode is only supported for running a command" >&2
```

Comment:
(reviewer commentary) I wondered if we were still using `--mode build` but it was just hidden slightly by the diff on the lines above this.